.github/labeler.yml

```
@@ -0,0 +1,57 @@
+- changed-files:
+  - any-glob-to-any-file: /charts/ad2image/**
+
+appswitcher-server:
+- changed-files:
+  - any-glob-to-any-file: /charts/appswitcher-server/**
+
+java-spring-boot:
+- changed-files:
+  - any-glob-to-any-file: /charts/java-spring-boot/**
+
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)